src/main/scripts/org/reaktivity/specification/nukleus/http/streams/rfc7230/connection.management/request.with.header.exclude/client.rpt

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright 2016-2020 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+connect await ROUTED_CLIENT
+        "nukleus://streams/http#0"
+    option nukleus:window 8192
+    option nukleus:transmission "half-duplex"
+
+write nukleus:begin.ext ${http:beginEx()
+                              .typeId(nukleus:id("http"))
+                              .header(":scheme", "http")
+                              .header(":method", "OPTIONS")
+                              .header(":path", "/")
+                              .header(":authority", "localhost:8080")
+                              .header("access-control-request-headers", "origin, x-custom-header")
+                              .build()}
+connected
+
+write close
+
+read nukleus:begin.ext ${http:beginEx()
+                             .typeId(nukleus:id("http"))
+                             .header(":status", "200")
+                             .header("content-length", "0")
+                             .header("access-control-allow-headers", "origin")
+                             .build()}
+
+read closed
+
+read notify PREFLIGHT_REQUEST_COMPLETED
+
+connect await PREFLIGHT_REQUEST_COMPLETED
+        "nukleus://streams/http#0"
+    option nukleus:window 8192
+    option nukleus:transmission "half-duplex"
+
+write nukleus:begin.ext ${http:beginEx()
+                              .typeId(nukleus:id("http"))
+                              .header(":scheme", "http")
+                              .header(":method", "GET")
+                              .header(":path", "/")
+                              .header(":authority", "localhost:8080")
+                              .header("x-custom-header", "")
```

Comment:
Recommend sending non-empty header in the example request, especially for the header that will be excluded otherwise one might infer it was excluded because it was empty.